adlfs/multithread.py

```diff
@@ -57,13 +58,17 @@ class ADLDownloader(object):
         by constructor.
     run: bool [True]
         Whether to begin executing immediately.
+    overwrite: bool [False]
+        Whether to forcibly overwrite existing files/directories.
```

Comment:
Please extend: if it's a directory, will quit if this is False, whether any files would be overwritten or not - only matching filenames are actually overwritten if True.